### PR TITLE
modified scaper to reflect new image file name

### PR DESCRIPTION
**Pull request by philkim72**
The new image file will look like this: 2019-04-20_18-17-12.jpg

### CODE REVIEW

File: scraper_service/main.py [hunk 1]
```diff
@@ -1,15 +1,20 @@
 import shutil
 from time import time
+from random import randint
```
Comment (ek-ok) [inline scraper_service/main.py]: Can we remove this line as `randint ` is not being used?

File: scraper_service/main.py [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
+from datetime import datetime, timezone
```
Comment (ek-ok) [inline scraper_service/main.py]: Can we remove this line as `timezone ` is not being used?

File: scraper_service/main.py [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
 
 import requests
 from fake_useragent import UserAgent
 
 
 def scrape_handler():
-    now = int(time())
+   
+    timestamp = datetime.now()
+    timestampString = str(timestamp.date()) + "_" + str(timestamp.hour).zfill(2) + "-" + str(timestamp.minute).zfill(2) \
```
Comment (ek-ok) [inline scraper_service/main.py]: You might find this convenient.
`timestamp_str = timestamp.strftime("%Y_%m_%d_%H%M_%S")`

Would it be ok to stick with `variable_with_under_score` format?
More info here https://pep8.org/

File: scraper_service/main.py (hunk 1 above)
Comment (philkim72) [reply scraper_service/main.py]: yes, let's remove it.

File: scraper_service/main.py (hunk 2 above)
Comment (philkim72) [reply scraper_service/main.py]: yes let's remove it.

File: scraper_service/main.py (hunk 3 above)
Comment (philkim72) [reply scraper_service/main.py]: thanks for the tip.  I will revise later tonight

**Pull request merged**